DeUrgenta/Features/CreateGroup/Coordinator/CreateGroupCoordinator.swift

```diff
@@ -14,6 +17,37 @@ final class CreateGroupCoordinator: Coordinator {
         let viewController = UIHostingController(rootView: CreateGroupView(delegate: self))
         navigationController.pushViewController(viewController, animated: true)
     }
+
+    func presentMessageCompose(withItem contact: ContactInfo) {
+        guard MFMessageComposeViewController.canSendText() else {
+            return
+        }
+        viewModel.currentContact = contact
+
+        let composeVC = MFMessageComposeViewController()
+        composeVC.messageComposeDelegate = self
+
+        var tempArray: [String] = []
+        tempArray.append(contact.phoneNumber?.stringValue ?? "")
+        composeVC.recipients = tempArray
```

Comment:
Usually, we should use better naming than `tempArray`, but for this case I think we have to guard first - if there is no phone number we should not present the message composer. 
If we have the number we can use something like composeVC.recipients = [phoneNumber]